### PR TITLE
Azure container app typo fix and add some validation

### PR DESCRIPTION
**Pull request by xwang971**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yonzhan** [comment]
containerapp

File: src/containerapp/azext_containerapp/_utils.py [hunk 1]
```diff
@@ -361,6 +361,12 @@ def parse_secret_flags(secret_list):
         identity_Id = ""
 
         kv_identity = value.split(',', 2)
+        if len(kv_identity) == 1:
```
Comment (sanchitmehta) [inline src/containerapp/azext_containerapp/_utils.py:364]: What if it starts with `keyvaultref` and `identityref` is not present?

Comment (xwang971) [reply src/containerapp/azext_containerapp/_utils.py:364]: Next line we will do the check and will return "Identityref is missing. Secrets must be in format \"<key>=<value> <key>=<value> ...\" or \"<key>=<keyvaultref:keyvaulturl,identityref:indentityId> ...\."

**Comment by xwang971** [comment]
@zhoxing-ms Can I get this reviewed/approved and have a new release kicked off please? Thanks!

**Comment by zhoxing-ms** [comment]
@xwang971 Do you need to release a new extension version for this PR? If so, please upgrade the version defined in the `setup,py`.

**Comment by zhoxing-ms** [comment]
@xwang971 Please resolve the confilcts

**Comment by xwang971** [comment]
> @xwang971 Please resolve the confilcts

@zhoxing-ms hi, conflicts are resolved and version has been upgraded. Thanks

**Review by zhoxing-ms — APPROVED** [review]

**Comment by zhoxing-ms** [comment]
@sanchitmehta Could you please help review this PR again?

**Review by jijohn14 — APPROVED** [review]

**Review by sanchitmehta — APPROVED** [review]

**Comment by sanchitmehta** [comment]
Approved, LGTM!

**Comment by xwang971** [comment]
@zhoxing-ms hi, can I please get this merged and new release cut

**Pull request merged**